.cargo/config.toml

```diff
@@ -27,3 +27,7 @@ rustflags = ["--cfg", "espidf_time64"]
 
 [unstable]
 build-std = ["std", "panic_abort"]
+
+[env]
+ESP_IDF_VERSION = "v5.4.1"
+BINDGEN_EXTRA_CLANG_ARGS = "-D CONFIG_I2C_BUS_BACKWARD_CONFIG"
```

Comment:
This appears to be unavoidable at the moment.

It would be great it if was something we can set in the module so the downstream user doesn't need to think about it, but haven't found a way to either

    set the env var from the dependency
    get embuild to recognize this as a [CInclArgs](https://docs.rs/embuild/latest/embuild/build/struct.CInclArgs.html).

if setting this in our template's .cargo/config.toml is harmless, it might be something to consider until we:

    have support for the newer i2c driver or
    figure out how to get this into the embuild system without stepping on esp-idf-sys's use of it

Reply:
@npmenard suggested including the sdkconfig header in the `bindings.h` of the bme280. Could you try that and see if it works.

Reply:
by sdkconfig header, do you mean the `i2c_bus`? 

I've experimented on a new rust idf-template including just the `i2c_bus` component and binding it, but will still not compile without the extra clang configs.

Reply:
let not have this define here and use the sdkconfig.h